frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -75,7 +75,7 @@ class DocumentList extends Component {
   constructor(props) {
     super(props)
 
-    this.state.selectedRows = {}
+    this.state = this.defaultLocalState()
```

Comment:
The logic for containers is the same, they should still be pretty lightweight in local state. Not sure what you mean by the logic behind setting that state, but I still think the best place to initialise the state is within the bounds of the constructor block, as it's common practice and expected behaviour. Whatever you prefer, though.